pkg/agent/clientset.go

```diff
@@ -43,7 +43,7 @@ type ClientSet struct {
 	// proxy server.
 	probeInterval time.Duration // The interval by which the agent
 	// periodically checks if its connections to the proxy server is ready.
-	dialOption grpc.DialOption
+	dialOption []grpc.DialOption
```

Comment:
Slightly more work but can we name it dialOtions to hint that its a slice?

Reply:
Updated.